Documentation/ExtensionArchitecture/FileStructure/ExtEmconf.rst

```diff
@@ -257,6 +256,12 @@ Example:
    :Data type:
          boolean
    :Description:
+         .. deprecated:: 12.1
+            When loading or unloading extensions using the extension manager,
+            all caches are always cleared. Extensions that want to keep
+            compatibility with both TYPO3 v11 and v12 should keep the setting
+            until v11 compatibility is dropped from the extensions.
+
          If set, the extension manager will request all caches (incl. frontend
```

Comment:
This text does not fit with the deprecation warning above

Reply:
That's the text from the changelog. I interpret it that way, that on v12 the cache is always cleared when installing a new extension, in v11 not.

Reply:
yes so the text below should be removed for v12

Reply:
It does not hurt and it is removed with v13 anyway from the docs.

Reply:
Anyway, I'll remove it, don't care much about it.

Reply:
The sentence above and this one are somehow contradicting

Reply:
Which sentence do you mean?

Reply:
"When loading or unloading extensions using the extension manager,
            all caches are always cleared." vs " If false (default),
         only the system cache will be cleared." The latter sentence is clearly wrong on v12

Reply:
Ah okay, yes, can be removed, you are right.
Don't know why it is deprecated and not completely removed, it is an array key which does not have any effect anymore. But anyway ;-)